src/sdk/Viewport3d.ts

```diff
@@ -403,7 +409,7 @@ export class Viewport3d implements ViewportDelegate {
       // Update the camera position relative to the player's mesh
       const v = new THREE.Vector3(0.5, 0, -0.5);
       v.add(actor.getModel().getWorldPosition());
-      this.pivot.position.lerp(v, 0.1);
+      this.pivot.position.lerp(v, 5 / Settings.fps);
```

Comment:
What's the thought process on this?

Reply:
Somewhat unrelated to this PR, I've been picking through my customized code. 
I was able to get a much smoother experience by matching the `Settings.fps` with my expected framerate (120 fps). However, this made the camera feel too stiff, and this was the remediation. Open to remove this change, but should be inconsequential.

Reply:
Sounds good. People with really low fps (20-30) reported the camera lagging a bit too far behind the camera. Looks better on my screen too (240fps)